src/pages/login/login.css

```diff
@@ -0,0 +1,37 @@
+.login {
+    --background: url(../../assets/Walktober-graphic.png) center top;
+}
+
+.orDivider{
```

Comment:
The standard for css classes is kebab case. Could we change these to .or-divider?

Reply:
Changed!